stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -387,6 +394,15 @@ extension Unsafe${Mutable}BufferPointer: ${Mutable}Collection, RandomAccessColle
 }
 
 extension Unsafe${Mutable}BufferPointer {
+  @_alwaysEmitIntoClient
+  internal init(
+    @_nonEphemeral _uncheckedStart start: Unsafe${Mutable}Pointer<Element>?,
+    count: Int
+  ) {
+    _position = start
+    self.count = count
+  }
+
```

Comment:
Should this also be `@inlinable`? I don't think AEIC implies that, but I don't recall the details.

What is `@_nonEphemeral`?

Reply:
`@_aeic` does imply `@inlinable` -- indeed, it is a stronger variant of it. 

The `@_nonEphemeral` attribute indicates that the associated function escapes the pointer. The compiler emits a warning when a temporary pointer (such as the ones resulting from the implicit inout-to-pointer conversion feature) is passed for a parameter that has this attribute. It's a nice (if limited) safety feature for a common issue.